sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,14 +1,25 @@
 # Release History
 
 ## 1.0.0-beta.7 (Unreleased)
+### Added
+- Support real time notifications with new methods in ChatClient/ChatAsyncClient: 
+    - startRealtimeNotifications
+    - stopRealtimeNotifications
+    - on(chatEventId, listenerId, listener) 
+    - off(chatEventId, listenerId) 
+- Add a sample chat app under folder samples for testing and playing around chat functionality purpose
+- Add signaling folder and classes to support real time notification
```

Comment:
We should only mention things that affect the developers. 👇🏻 is an implementation detail. 
> Add signaling folder and classes to support real time notification

The main point is adding real-time notification that you have already mentioned in changelog